### PR TITLE
adding angular clipboard

### PR DESCRIPTION
**Pull request by bradfordwagner**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] The package does not provide its own types, and you can not add them.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If adding a new definition:
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Run `tsc` without errors.
- [x] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header.

adding new definition

### CODE REVIEW

**Comment by bradfordwagner** [comment]
@dt-bot ready for review

File: angular-clipboard/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Type definitions for angular-clipboard v1.5
+// Project: https://github.com/omichelsen/angular-clipboard
+// Definitions by: Bradford Wagner <https://github.com/bradfordwagner/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'angular-clipboard' {
```
Comment (vvakame) [inline angular-clipboard/index.d.ts]: could you strip `declare module 'angular-clipboard' {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred

File: angular-clipboard/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "files": [
+    "index.d.ts",
+    "angular-clipboard-tests.ts"
+  ],
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "noImplicitAny": true,
+    "strictNullChecks": false,
```
Comment (vvakame) [inline angular-clipboard/tsconfig.json]: could you try to use `"strictNullChecks": true,`?

**Comment by yuit** [comment]
@bradfordwagner  thank you for the PR, could you address @vvakame 's comment

**Comment by bradfordwagner** [comment]
@yuit @vvakame I've addressed the comments thank you for taking the time to review.

**Pull request merged**